Doc/Makefile

```diff
@@ -107,7 +107,7 @@ clean:
 
 venv:
 	$(PYTHON) -m venv venv
-	./venv/bin/python3 -m pip install -U Sphinx
+	./venv/bin/python3 -m pip install sphinx~=1.5.6
```

Comment:
Keeping version details out of the Makefile sounds like a better option to me. Makefiles can increase the complexity of its own :)